pkgs/applications/graphics/renderdoc/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, cmake, qtbase, qtx11extras, git
+, pkgconfig, xorg, makeQtWrapper, vulkan-loader
+}:
+
+stdenv.mkDerivation rec {
+  name = "renderdoc-${version}";
+  version = "0.34pre";
+
+  src = fetchFromGitHub {
+    owner = "baldurk";
+    repo = "renderdoc";
+
+    rev = "5e2717daec53e5b51517d3231fb6120bebbe6b7a";
+    sha256 = "1zpvjvsj5c441kyjpmd2d2r0ykb190rbq474nkmp1jk72cggnpq0";
+  };
+
+  buildInputs = [ cmake qtbase git pkgconfig xorg.libpthreadstubs xorg.libXdmcp qtx11extras ];
```

Comment:
cmake, pkgconfig would go into nativeBuildInputs. Git should not be specified as a dependency